components/style/properties/longhand/box.mako.rs

```diff
@@ -594,13 +594,21 @@ ${helpers.single_keyword("overflow-x", "visible hidden scroll auto",
                             p2y = try!(specified::parse_number(input));
                             Ok(())
                         }));
+                        if (p1x < 0.0 || p1x > 1.0 ) || (p2x < 0.0 || p2x > 1.0)  {
+                            return Err(())
+                        }
+
                         let (p1, p2) = (Point2D::new(p1x, p1y), Point2D::new(p2x, p2y));
                         Ok(SpecifiedValue::CubicBezier(p1, p2))
                     },
                     "steps" => {
                         let (mut step_count, mut start_end) = (0, StartEnd::End);
                         try!(input.parse_nested_block(|input| {
                             step_count = try!(specified::parse_integer(input));
+                            if step_count < 0 {
```

Comment:
From reading the spec:

> It must be a positive integer greater than 0.

So zero is invalid too. Please fix that and add a test.